modules/build/src/main/scala/scala/build/bsp/BspServer.scala

```diff
@@ -145,6 +147,10 @@ class BspServer(
       scala.build.blooprifle.internal.Constants.bspVersion,
       capabilities
     )
+    val intellijBSPName        = "IntelliJ-BSP"
+    val buildComesFromIntelliJ = params.getDisplayName == "IntelliJ-BSP"
```

Comment:
IMHO we should check if `Intellij` is present in the name (case insensitive) - it should be a bit more roboust

Reply:
fixed